### ISSUE TITLE
Wireframe for GIS team

### ISSUE THREAD

**Issue by mcharg** [ONRR/nrrd#1908]

**Comment by jkwonatx** [comment]
https://4ipg77.axshare.com/#id=wd5tip&p=page_1

@mcharg I hope I'm on the right track. On the file above, you will find two wireframes and one set of screenshot images. 

The first wireframe is a Reporting page I copied from our new ONRR site. The second wireframe is the actual deliverable for the GIS team. The set of images is what they currently have to work off of. 

For the deliverable (the second wireframe) I'm simply replacing the ONRR content with GIS's content. 

I'd like to know whether you think this should be enough to get the discussion going at our meeting today. I do have a bit more to touch up. Thank you.

**Comment by jkwonatx** [comment]
FYI, Made updates to the axure file above since this morning.

**Comment by mcharg** [comment]
@jkwonatx I think this is a great starting point for discussion.

**Comment by jkwonatx** [comment]
made updates

https://60olxm.axshare.com

**Comment by mcharg** [comment]
@jkwonatx I added comments in Axure.

**Comment by jkwonatx** [comment]
@mcharg I'm making edits to the GIS wireframe. I checked the ONRR prototype to get the dimension info and it seems to say  that it's in 389:150 ratio. So I entered those numbers to the Height and Width and expanded the photo proportionately (shift+drag). I want to make sure this is correct. Thank you. 

![Screen Shot 2022-01-27 at 1 18 37 PM](https://user-images.githubusercontent.com/85380376/151428515-c32d022b-3edd-418b-865c-651dd4f915fb.png)
